### PR TITLE
feat: Submission flow tracking

### PR DESCRIPTION
**Pull request by olerichter00**
This PR resolves [ONYX-953] 

- Cohesion PR: https://github.com/artsy/cohesion/pull/503

### Description

This PR adds tracking to the new submission flow. The list of events can be found [here](https://docs.google.com/spreadsheets/d/1ZTCKU04K-dArhoq48Z3e0ptGuhU6N5cvxSgehrh28jQ/edit#gid=0).

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-953]: https://artsyproduct.atlassian.net/browse/ONYX-953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0ee9dbf747484f7a026599156fe8042028efe55e
</p>

File: src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAddDetails.tsx [hunk 1]
```diff
@@ -18,48 +21,52 @@ export const SubmitArtworkAddDetails = () => {
   ).current
 
   return (
-    <Flex px={2} flex={1}>
-      <ScrollView>
-        <Text variant="lg-display" mb={2}>
-          Artwork details
-        </Text>
+    <ProvideScreenTrackingWithCohesionSchema
+      info={screen({ context_screen_owner_type: OwnerType.conversation })}
```
Comment (MounirDhahri) [inline src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAddDetails.tsx:25]:
```suggestion
      info={screen({ context_screen_owner_type: OwnerType.artworkFormAddDetails })}
```

**Review by MounirDhahri — DISMISSED** [review]
Awesome! just one minor comment

File: src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAddDetails.tsx (hunk 1 above)
Comment (olerichter00) [reply src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAddDetails.tsx]: Thanks, good catch!

**Pull request merged**